07-routing-gossip.md

```diff
@@ -262,6 +262,14 @@ nodes not associated with an already known channel are ignored.
    * [`32*byte`:`alias`]
    * [`u16`:`addrlen`]
    * [`addrlen*byte`:`addresses`]
+   * [`node_ann_tlvs`:`tlvs`]
+
+1. `tlv_stream`: `node_ann_tlvs`
+2. types:
+   1. type: 1 (`option_will_fund`)
+   2. data:
+       * [`lease_rates`:`lease_rates`]
```

Comment:
How would you set `funding_weight` here?
It feels like a very dynamic value that would depend on `requested_sats` and your utxo set at channel open time, it's really hard to choose a value that would fit all requests...

Reply:
The idea is that you pick a value that you want the opening peer to pay you for -- they're not going to pay you for more than that, input aggregation you should do on your own dime ;)

c-lightning defaults to charging for the weight of two P2WPKH inputs and a P2WPKH output.